android.js

```diff
@@ -103,13 +107,25 @@ async function signAndroidApp() {
 		log("started zipalign")
 
 		// Android requires all resources to be aligned for mmap. Must be done.
-		execFileSync(`${androidHome}/build-tools/${BUILD_TOOLS_V}/zipalign`, [
+		exec(`${androidHome}/build-tools/${BUILD_TOOLS_V}/zipalign`, [
 			'4',
 			'app-android/' + apkPath,
 			outPath
 		])
 	} else {
-		execFileSync('mv', ['app-android/' + apkPath, outPath])
+		exec('mv', ['app-android/' + apkPath, outPath])
 	}
 	log(`APK was moved to\n${outPath}`)
 }
+
+function exec(...args) {
```

Comment:
I would rename it because at first I thought it's node's `exec` and it doesn't look like correct usage of it.
I also don't like to throw non-error types but maybe it's okay.